typegraph/core/src/runtimes.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
+// SPDX-License-Identifier: MPL-2.0
+
+use crate::{
+    global_store::store,
+    wit::core::RuntimeId,
+    wit::runtimes::{self as wit, Error as TgError},
+};
+
+type Result<T, E = TgError> = std::result::Result<T, E>;
+
+#[derive(Debug)]
+pub enum Runtime {
+    Deno,
+}
+
+#[derive(Debug)]
```

Comment:
should not this file be merged with common at one point? or some of the definitions?

Reply:
Yes. I think most of the types should be shared. But for now, I do not want to change a lot of things in `common`. I would be a lot easier to merge the we add more runtime supports in the new typegraph sdk.